src/components/Dashboard/AddressListSm.vue

```diff
@@ -1,15 +1,19 @@
 <template>
   <div id="address-list-sm">
+    <template v-if="isFirm">
+        <firms-address-list
```

Comment:
If you use the PascalCase name here then VS Code lets you click F12 to display the component.

Reply:
PascalCase starts with a capital.

But I see it works in camelCase too :)

Reply:
Oh right :P . Updating it